package.json

```diff
@@ -3,11 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@fontsource/roboto": "^4.5.8",
```

Comment:
I think the font for this project should be inter, not roboto

<img width="1533" alt="Screenshot 2022-11-18 at 10 26 28 p m" src="https://user-images.githubusercontent.com/3700448/202834052-dd1c6be8-c862-45b7-8eb4-f2c1f59a30e9.png">

Reply:
ok to change it